src/Components/FloorSwitcher/FloorSwitcher.jsx

```diff
@@ -116,53 +114,53 @@ class FloorSwitcher extends PureComponent {
 
-  renderMobilePicker(floors) {
-    const { activeFloor } = this.props;
-    let selectedFloor = activeFloor;
-
-    if (!selectedFloor) {
-      // choose floor "0" or first floow
-      selectedFloor = floors.find(f => f.iabp_name === '0') || floors[0];
-    }
-
-    // find previous and next floor
-    let prevFloor = null;
-    let nextFloor = null;
-
-    for (let i = 0; i < floors.length; i += 1) {
-      if (floors[i] === selectedFloor) {
-        prevFloor = i > 0 ? floors[i - 1] : null;
-        nextFloor = i < floors.length - 1 ? floors[i + 1] : null;
-        break;
-      }
-    }
-
-    const prevButtonClass = `tm-prev-floor ${prevFloor ? '' : 'disabled'}`;
-    const nextButtonClass = `tm-next-floor ${nextFloor ? '' : 'disabled'}`;
-
-    return (
-      <div className="tm-floor-switcher-mobile">
-        <Button
-          className={`tm-floor-button-mobile ${prevButtonClass}`}
-          onClick={() => prevFloor && this.selectFloor(prevFloor)}
-        >
-          <FiArrowUp />
-        </Button>
-        <FloorButton
-          active={!!activeFloor}
-          floor={selectedFloor}
-          isMobile
-          onClick={f => this.selectFloor(f)}
-        />
-        <Button
-          className={`tm-floor-button-mobile ${nextButtonClass}`}
-          onClick={() => nextFloor && this.selectFloor(nextFloor)}
-        >
-          <FiArrowDown />
-        </Button>
-      </div>
-    );
-  }
+  // renderMobilePicker(floors) {
+  //   const { activeFloor } = this.props;
+  //   let selectedFloor = activeFloor;
+
+  //   if (!selectedFloor) {
+  //     // choose floor "0" or first floow
+  //     selectedFloor = floors.find((f) => f.iabp_name === '0') || floors[0];
+  //   }
+
+  //   // find previous and next floor
+  //   let prevFloor = null;
+  //   let nextFloor = null;
+
+  //   for (let i = 0; i < floors.length; i += 1) {
+  //     if (floors[i] === selectedFloor) {
+  //       prevFloor = i > 0 ? floors[i - 1] : null;
+  //       nextFloor = i < floors.length - 1 ? floors[i + 1] : null;
+  //       break;
+  //     }
+  //   }
+
+  //   const prevButtonClass = `tm-prev-floor ${prevFloor ? '' : 'disabled'}`;
+  //   const nextButtonClass = `tm-next-floor ${nextFloor ? '' : 'disabled'}`;
+
+  //   return (
+  //     <div className="tm-floor-switcher-mobile">
+  //       <Button
+  //         className={`tm-floor-button-mobile ${prevButtonClass}`}
+  //         onClick={() => prevFloor && this.selectFloor(prevFloor)}
+  //       >
+  //         <FiArrowUp />
+  //       </Button>
+  //       <FloorButton
+  //         active={!!activeFloor}
+  //         floor={selectedFloor}
+  //         isMobile
+  //         onClick={(f) => this.selectFloor(f)}
+  //       />
+  //       <Button
+  //         className={`tm-floor-button-mobile ${nextButtonClass}`}
+  //         onClick={() => nextFloor && this.selectFloor(nextFloor)}
+  //       >
+  //         <FiArrowDown />
+  //       </Button>
+  //     </div>
+  //   );
+  // }
```

Comment:
not needed anymore ?

Reply:
I have no idea, but isAppWidthSmallerThanS has been commented out for a while, so the whole render method is useless. I dunno, maybe we still need it in the future